config/config.exs

```diff
@@ -1,6 +1,13 @@
 import Config
 
 config :geo,
-  maxmind_license_key: System.get_env("MAXMIND_LICENSE_KEY")
+  maxmind_license_key:
+    System.get_env("MAXMIND_LICENSE_KEY") ||
```

Comment:
I guess [`System.fetch_env!/1`](https://hexdocs.pm/elixir/1.13/System.html#fetch_env!/1) could also have worked, but would be less explicit.